test/functional/org_admin/templates_controller_test.rb

```diff
@@ -189,26 +189,24 @@ class TemplatesControllerTest < ActionDispatch::IntegrationTest
 
     # We shouldn't be able to edit a historical version
     put org_admin_template_path(prior), {template: params}
-    assert_equal _('You can not edit a historical version of this template.'), flash[:notice]
-    assert_response :redirect
-    assert_redirected_to edit_org_admin_template_url(prior)
-    assert assigns(:template)
+    assert_response :forbidden
+    json_body = ActiveSupport::JSON.decode(response.body)
+    puts json_body.inspect
```

Comment:
looks good Jose. just remove this leftover debug statement

Reply:
good catch!